src/main/java/stsjorbsmod/patches/CoupDeGracePatch.java

```diff
@@ -0,0 +1,61 @@
+package stsjorbsmod.patches;
+
+import com.evacipated.cardcrawl.modthespire.lib.SpirePatch;
+import com.evacipated.cardcrawl.modthespire.lib.SpirePrefixPatch;
+import com.evacipated.cardcrawl.modthespire.lib.SpireReturn;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.IntangiblePlayerPower;
+import stsjorbsmod.powers.CoupDeGracePower;
+
+public class CoupDeGracePatch {
```

Comment:
For the monster intent, you might be able to get away with patching `AbstractMonster::calculateDamage` such that you wrap the existing `this.intentDmg = dmg;` with something like 
```
if shouldCalculatePreentedDamage
  intentDmg = 1
else
  proceed
```

Reply:
Ohh, that's a path I hadn't considered. I'll look into this.

Reply:
I'm curious if you thought more about this. _Not that I quite remember what I was thinking_ lol